lib/logstash/pluginmanager/base.rb

```diff
@@ -0,0 +1,37 @@
+class LogStash::PluginManager::Base < Clamp::Command
+  def gemfile
+    @gemfile ||= LogStash::Gemfile.new(File.new(LogStash::Environment::GEMFILE_PATH, 'r+')).load
+  end
+
+  # If set in debug mode we will raise an exception and display the stacktrace
+  def report_exception(readable_message, exception)
+    if ENV["DEBUG"]
```

Comment:
Technically both raise and exception, but `signal_error` exception are pretty printed by clamp..
As a developer I want to have to the real stacktrace.